examples/foomedical/src/main.tsx

```diff
@@ -40,6 +41,7 @@ root.render(
     <BrowserRouter>
       <MedplumProvider medplum={medplum}>
         <MantineProvider theme={theme}>
+          <Notifications />
```

Comment:
This was missing so notifications were not showing. For some reason the build is now breaking with the following error, I'll investigate what's going on.
```
Error: src/main.tsx(2,10): error TS2300: Duplicate identifier 'Notifications'.
Error: src/main.tsx(4,10): error TS2300: Duplicate identifier 'Notifications'.
```